packages/patterns/src/keys/checkKey.js

```diff
@@ -392,48 +391,14 @@ harden(getCopyMapValues);
  * @template {Key} K
  * @template {Passable} V
  * @param {CopyMap<K,V>} m
- * @returns {Array<[K,V]>}
- */
-export const getCopyMapEntryArray = m => {
```

Comment:
TODO test in agoric-sdk

I renamed this method with its types to `getCopyMapEntries` which is more inline with similar names for other copy collections. I deleted the old `getCopyMapEntries` which returned an iterable, because nothing in endo or agoric-sdk seems sensitive to the difference between this being only an Iterable vs it being an array that also happens to be an Iterable. The name `getCopyMapEntryArray` was only used within this package and was never exported beyond this package, so no need to deprecate. We simply removed it.